pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -245,6 +250,14 @@ func (b *Build) BuildPod(images BuildPodImages, buildContext BuildContext) (*cor
 		Resources: b.Spec.Resources,
 		Args: args(
 			[]string{"-layers=/layers", "-analyzed=/layers/analyzed.toml"},
+			func() []string {
+				if !platformAPILessThan07 {
+					if b.Spec.RunImage.Image != "" {
```

Comment:
Maybe we should drop platform api support

Reply:
below 0.7?